master/getting-started/kubernetes/installation/migration-from-flannel.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Migrating a cluster from flannel to Calico
+---
+
+# About
+
```

Comment:
The article describes how to migrate an existing Kubernetes cluster using Flannel networking to use Calico
for networking.